app_server/app/Router.js

```diff
@@ -97,6 +97,9 @@ server.register(require('inert'), function(err) {
   server.route({
     method: 'GET',
     path: '/admin/{param*}',
+    config: {
+      auth: false
+    },
```

Comment:
The user must be able to access the page to even log in (aside from sending a post request directly).
The auth should be checked on API request, rather than on the static file handler.
Should `auth` be something else to accomplish that?